store/tikv/rawkv.go

```diff
@@ -273,6 +273,7 @@ func (c *RawKVClient) DeleteRange(startKey []byte, endKey []byte) error {
 // If you want to exclude the startKey or include the endKey, append a '\0' to the key. For example, to scan
 // (startKey, endKey], you can write:
 // `Scan(append(startKey, '\0'), append(endKey, '\0'), limit)`.
+// Scanning from the last element is not supported.
```

Comment:
Emmmm, I didn't mean this. Scanning in forward order from the end makes nonsense. I meant, your code doesn't supports `ReverseSeek` from `""`, which means, scanning from the end in backward order. That's because when it tries to LocateEndKey(`""`), it only got nil. You can try to make it return the last region in this case. Or if it's difficult to do this, add a comment to the function `ReverseSeek` to explain that.

Reply:
Where is ReverseSeek? Is it tikv?

Reply:
Oh, sorry, I meant, `ReverseScan`